common/net/credentials/alts/handshaker/handshaker.go

```diff
@@ -239,10 +264,10 @@ func (s *altsHandshaker) ClientHandshake(_ context.Context) (net.Conn, credentia
 	}
 
 	// do the client handshake
-	if err := s.doClientHandshake(); err != nil {
+	if err := s.doClientHandshake(ctx, sign, verify, signInfo); err != nil {
```

Comment:
should group sign, verify into one object

Reply:
will fix in next commit